doc/changelogs/CHANGELOG_V23.md

```diff
@@ -40,6 +41,75 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="23.3.0"></a>
+
+## 2024-11-20, Version 23.3.0 (Current), @RafaelGSS
+
+### Notable Changes
+
+* \[[`5767b76c30`](https://github.com/nodejs/node/commit/5767b76c30)] - **doc**: enforce strict policy to semver-major releases (Rafael Gonzaga) [#55732](https://github.com/nodejs/node/pull/55732)
+* \[[`ccb69bb8d5`](https://github.com/nodejs/node/commit/ccb69bb8d5)] - **(SEMVER-MINOR)** **src**: add cli option to preserve env vars on dr (Rafael Gonzaga) [#55697](https://github.com/nodejs/node/pull/55697)
```

Comment:
When working on the blog post, IMO it would be useful to amend this to say "diagnostic report" instead of "dr"

Reply:
I agree, but I also think that we should not accept "dr" in the commit message in the first place.

Reply:
I wrote it to not exceed the commit message length.